modules/TestUtils.js

```diff
@@ -28,6 +28,12 @@ export const isArray = (object) =>
   Array.isArray(object)
 
 /**
+ * Returns true if the given object is a set.
+ */
+export const isSet = (object) =>
+  object instanceof Set
```

Comment:
You'd need to cache the getter from `Set.prototype.size`, and `.call` it in a `try`/`catch`, in order to reliably detect `Set`s.